guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc

```diff
@@ -0,0 +1,83 @@
+[id="Exporting_a_Content_View_Version_Syncable_{context}"]
+= Exporting a Content View Version in a Syncable Format
```

Comment:
Please adjust the _anchor and file name_ to match the title aka `[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]`